src/main/java/tay/resource/MessageReceiveResource.java

```diff
@@ -0,0 +1,58 @@
+package tay.resource;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Request;
+import javax.ws.rs.core.Response;
+
+import com.twilio.twiml.Body;
+import com.twilio.twiml.Message;
+import com.twilio.twiml.MessagingResponse;
+import com.twilio.twiml.TwiMLException;
+
+import tay.utils.MessageUtils;
+
+
+
+@Path("receive")
+public class MessageReceiveResource {
+
+	@POST
+	public Response textMe(String request) throws ServletException {
```

Comment:
33-39 are to output the message received result into a file. Changes it into a Hash first so it's easy to read, have to do some simple text parsing. 

41-43 Builds a Twilio message object. 

45, builds a MessageResponse Twilio object from the message object. 

46-52, returns an OK HTTP response as the Twilio MessageResponse object turned to xml. per twilio's api. Responses have to be returned as xml.